api/src/utils/auth-utils.js

```diff
@@ -150,3 +150,29 @@ function verifySecret(currentScopes, tokenString, secret, req, callback, sendErr
     }
   });
 }
+
+/**
+ * Checks if a user has a role against an array of valid roles.
+ * 
+ * @param {Array<string>|string} validRoles Roles to search for a match.
+ * @param {Array<string>|string} userRoles Roles to match against.
+ * @returns {boolean} Indication if a match is found.
+ */
+exports.userInRole = function(validRoles, userRoles) {
```

Comment:
Petty little quibble. Rename function to `userInRoles` as it accepts an array of validRoles, vs. a single role to check.